routing/car_router.cpp

```diff
@@ -578,6 +581,26 @@ bool CarRouter::DoesEdgeIndexExist(Index::MwmId const & mwmId)
   return true;
 }
 
+bool CarRouter::AllMwmsHasRoutingIndex() const
+{
+  int64_t constexpr kFirstVersionWithRoutingIndex = 161206;
```

Comment:
Please add a detailed comment for this constant. It could be useful in the future.